packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/iam/iam.html

```diff
@@ -0,0 +1,58 @@
+<div class="text-center" data-ng-if="$ctrl.loading">
+    <oui-spinner></oui-spinner>
+</div>
+
+<div class="row" data-ng-if="!$ctrl.loading">
+    <div class="col-sm-9">
+        <div data-translate="dedicatedCloud_USER_iam_description"></div>
+        <div>
+            <small
+                data-translate="dedicatedCloud_USER_iam_usage_warning"
+            ></small>
+        </div>
+        <dl>
+            <dt data-translate="dedicatedCloud_USER_iam_status"></dt>
+            <dd>
+                <ovh-manager-pcc-dashboard-component-activation-status
+                    class="d-block mt-1"
+                    data-status-name=":: $ctrl.iamStatus.state"
+                ></ovh-manager-pcc-dashboard-component-activation-status>
+            </dd>
+        </dl>
+    </div>
+    <div class="col-sm-3">
+        <button
+            type="button"
+            class="mb-3 btn btn-block btn-default"
+            data-ng-if="$ctrl.iamEnabled || $ctrl.iamDisabled"
```

Comment:
**nitpick:** This is still needed? Can `iam` have any other status apart from `enabled` or `disabled`?
```suggestion
```